plugins/plugin-upload/frontend/app.js

```diff
@@ -342,7 +343,7 @@ function validate_reset() {
                     var logpath = path.resolve(__dirname, './../../../log/plugins-disable' + (new Date().toISOString().replace('T', ':')) + '.log');
 
                     var mydir = path.resolve(__dirname + '/../scripts');
-                    run_command('bash ' + mydir + '/disable_plugins.sh ' + pluginlist.join(' '),
+                    run_command('bash', [mydir + '/disable_plugins.sh', ...pluginlist],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['pluginlist' used out of scope.](https://app.codacy.com/app/ar2rsawseen/countly-server/pullRequest?prid=4577914)